app/views/admin/events/_action_page_events.html.erb

```diff
@@ -16,21 +16,23 @@
     <hr />
     <div class="row">
       <div class="col-sm-12">
-        <h3>Action Page Views</h3>
-        <%= line_chart(admin_action_page_events_path(@actionPage, type: :views),
-                       height: "200px", discrete: true) %>
+        <% Ahoy::Event.action_types(@actionPage).each do |type| %>
+          <h3><%= humanize_event_type(type) %></h3>
```

Comment:
Not a merge blocker, but `humanize_event_type` looks a lot like the built-in `t` helper. It might be more conventional here to call out to that instead. You'd just need to add something like

```ruby
  ahoy:
    views: Action Page Views
    emails: Email Actions
    congress_messages: Congress Message Actions
```

to en.yml and call `t(type, scope: :ahoy)`.

Reply:
nice!